Makefile

```diff
@@ -7,8 +7,6 @@ endif
 
 COMMIT := $(shell git rev-parse --short HEAD)
 ldflags = "-X "github.com/ipfs/go-ipfs/repo/config".CurrentCommit=$(COMMIT)"
```

Comment:
Why do you keep those 2 lines here if you don't use them in this Makefile anymore?